shell/platform/android/test_runner/build.gradle

```diff
@@ -46,7 +46,8 @@ android {
     testImplementation "androidx.window:window-java:1.0.0-beta04"
     testImplementation "com.google.android.play:core:1.8.0"
     testImplementation "com.ibm.icu:icu4j:69.1"
-    testImplementation "org.mockito:mockito-core:3.11.2"
+    testImplementation "org.mockito:mockito-core:3.12.4"
+    testImplementation "org.mockito:mockito-inline:3.12.4"
```

Comment:
consider bumping these dependencies in a separate PR, so errors don't get mixed with errors caused by other changes.